app/views/layouts/application.html.erb

```diff
@@ -26,7 +26,11 @@
           <% end %>
           <% if  current_user && current_user.admin?%>
           <div class="align-cart">
-          <%= link_to "View Account", root_path, class: "link account"%>
+          <%= link_to "View Account", admin_dashboard_path, class: "link account"%>
+          <% elsif current_user %>
+          Cart: <%= @cart.total_count %>
+          <%= link_to "View Cart", '/cart',  class:"view_cart" %>
+            <%= link_to "View Account", orders_path, class: "link account"%>
```

Comment:
@Tyjoo26 @erikaannesmith lets point View Account to user show page. and have user show page contain a link for orders index

Reply:
Made the requested modifications, pushing the edits